packages/gasket-log/server.js

```diff
@@ -149,16 +135,16 @@ class Log {
     //
     // Create proxies per loglevel to each method in winston.
     //
-    Object.keys(config.syslog.levels).forEach(level => {
-      this[level] = winston[level].bind(this.winston);
+    Object.keys(levels).forEach(level => {
+      this[level] = winston[level].bind(winston);
```

Comment:
So, I just realized a potential problem with custom log levels... several of our plugins are pre-wired with logging at certain levels. What happens with `warning` [for example](https://github.com/godaddy/gasket/blob/6c37347cfe372506ea21af8ab340f7b1ac15c05f/packages/gasket-plugin-nextjs/config.js#L19)) if custom log levels are set to npm levels, which has `warn` instead?

Reply:
Great point. Maybe better if these merged in with the default logging levels instead or with a set of expected logging levels.

Reply:
Possibly? Would merging throw off prioritization though? Referencing [these levels](https://www.npmjs.com/package/winston#logging-levels), syslog `warning` is 4, but that's `verbose` for npm levels. Not sure that matters.

Another thought is maybe our plugins use a select that that must be provided, or can be auto resolved. For example `warning` could resolve to `warn`, but your custom log levels _must_ provide one or the other.

Reply:
We could, however, just make sure every config.syslog.level is included in `levels` if there's a `levels` provided and just throw with a good message if there's something missing, then the caller is forced to decide what they want their various levels to mean.

Reply:
I went with the "make sure the expected levels are defined" approach and put that in 47c6f2d